pass.el

```diff
@@ -6,7 +6,7 @@
 ;;         Damien Cassou <damien@cassou.me>
 ;; Version: 1.7
 ;; GIT: https://github.com/NicolasPetton/pass
-;; Package-Requires: ((emacs "24") (password-store "0.1") (f "0.17"))
+;; Package-Requires: ((emacs "24") (password-store "0.1") (password-store-otp "0.1.1") (f "0.17"))
```

Comment:
please update Emacs dependency to 24.3 as this is the version introducing user-error.